test/Npgsql.Tests/ReaderTests.cs

```diff
@@ -1138,8 +1138,12 @@ public async Task GetStream([Values(true, false)] bool isAsync)
 
                     using (var stream = await streamGetter(reader, 0))
                     {
-                        Assert.That(stream.CanSeek, Is.EqualTo(Behavior == CommandBehavior.Default));
-                        Assert.That(stream.Length, Is.EqualTo(expected.Length));
+                        Assert.That(stream.CanSeek, Is.EqualTo(!IsSequential));
+                        if (IsSequential)
+                            Assert.That(() => stream.Length, Throws.Exception.TypeOf<NotSupportedException>(), "Stream does not support seeking.");
```

Comment:
This looks like a behavior change... shouldn't sequential streams still support reading the Length? After all, we have it from the PG wire protocol...

Reply:
According to [Stream docs](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.length?view=netcore-3.1), the `Length` property should throw a `NotSupportedException` when the stream cannot seek, what is the case in sequential mode.